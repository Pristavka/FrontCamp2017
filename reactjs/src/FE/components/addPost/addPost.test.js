import React from 'react';
import AddPost from './addPost';

it('Should render a AddPost', () => {
  const wrapper = shallow(<AddPost />);
  expect(wrapper).toMatchSnapshot();
});

