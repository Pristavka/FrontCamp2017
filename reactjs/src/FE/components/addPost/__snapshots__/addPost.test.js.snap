// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a AddPost 1`] = `
<React.Fragment>
  <Header />
  <div
    className="formWrapper"
  >
    <h1>
      On this page you can add new post!
    </h1>
    <form
      onSubmit={[Function]}
    >
      <div>
        <h3>
          Author's name:
        </h3>
        <input
          name="author"
          onChange={[Function]}
          placeholder="Enter your name"
          required={true}
          type="text"
        />
      </div>
      <div>
        <h3>
          Enter post:
        </h3>
        <textarea
          name="post"
          onChange={[Function]}
          placeholder="Enter your post"
          required={true}
        />
      </div>
      <button
        type="submit"
      >
        Save post
      </button>
    </form>
  </div>
</React.Fragment>
`;
